src/init/features/dataconnect/index.ts

```diff
@@ -69,7 +69,9 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
 
   if (!config.has("dataconnect")) {
     config.set("dataconnect.source", dir);
-    config.set("dataconnect.location", info.locationId);
+  } else {
+    // Remove location if it was previously set.
+    config.set("dataconnect", { source: dir });
```

Comment:
Q: why this branch?

Isn't this equivalent to always do `config.set("dataconnect", { source: dir });`?

Reply:
Good point. Fixed.